Vagrantfile

```diff
@@ -2,7 +2,7 @@
 # vi: set ft=ruby :
 
 Vagrant.configure(2) do |config|
-  config.vm.box = "hashicorp/precise64"
+  config.vm.box = "bento/ubuntu-16.04"
```

Comment:
I guess this is preferable to https://atlas.hashicorp.com/ubuntu/boxes/xenial64 ?